data/codes.json

```diff
@@ -158,10 +158,30 @@
             "reason": "User has not verified his email-address"
         },
         "PCE032": {
-        "code": "PCE032",
-        "message": "PCE032 : Something went wrong while updating user's badge",
-        "reason": "Internal Server Error"
-        }
+            "code": "PCE032",
+            "message": "PCE032 : Something went wrong while updating user's badge",
+            "reason": "Internal Server Error"
+        },
+        "PCE036": {
+            "code": "PCE036",
+            "message": "No activity-level data was found",
+            "reason": "User is not logged in or his data in not logged in levelProgress Table"
+      },
+        "PCE037": {
+            "code": "PCE037",
+            "message": "Something went wrong while fetching level-progress data",
+            "reason": "Internal Server Error"
+      },
+        "PCE038": {
+            "code": "PCE038",
+            "message": "Something went wrong while updating level-progress data",
+            "reason": "Internal Server Error"
+      },
+        "PCE039": {
+            "code": "PCE039",
+            "message": "Could not update user's activity level",
+            "reason": "Request body was empty"
+      }
     },
 
```

Comment:
I think this line can be removed.

Reply:
But like all the Error Codes have followed this pattern until now. So, this may not look consistent.

Reply:
no no, I meant the empty line in middle

Reply:
line 186

Reply:
Done.